src/core/MOM_dynamics_split_RK2.F90

```diff
@@ -860,6 +881,20 @@ subroutine step_MOM_dyn_split_RK2(u, v, h, tv, visc, Time_local, dt, forces, p_s
   if (CS%id_u_BT_accel > 0) call post_data(CS%id_u_BT_accel, CS%u_accel_bt, CS%diag)
   if (CS%id_v_BT_accel > 0) call post_data(CS%id_v_BT_accel, CS%v_accel_bt, CS%diag)
 
+  ! Diagnostics for terms mutiplied by fractional thicknesses
+  if (CS%id_hfPFu > 0) then
+    do k=1,nz ; do j=js,je ; do I=Isq,Ieq
```

Comment:
These diagnostics aren't at q-points. Use the `is` and `js` instead

Reply:
CS%PFu is at u points and CS%PFv is at v points. So, these are at q points, right? I am following the approach of the piece of code in  lines
```
  do k=1,nz
    do j=js,je ; do I=Isq,Ieq
      u_bc_accel(I,j,k) = (CS%CAu(I,j,k) + CS%PFu(I,j,k)) + CS%diffu(I,j,k)
    enddo ; enddo
    do J=Jsq,Jeq ; do i=is,ie
      v_bc_accel(i,J,k) = (CS%CAv(i,J,k) + CS%PFv(i,J,k)) + CS%diffv(i,J,k)
    enddo ; enddo
  enddo
```

Reply:
Hrmm that's a little weird to me as well as to why the accelerations are using the q-point indices. I think they probably end up being the the 'correct ones' but maybe that's something to clarify on the Slack channel.

Reply:
Indices for acceleration terms should match du/dt and dv/dt points, which are q points, right?

Reply:
q-points should be where the divergence or curl is calculated: 
https://mom6.readthedocs.io/en/dev-gfdl/api/generated/pages/Horizontal_indexing.html

Reply:
Sorry, I mixed u/v-points with q points. We want acceleration terms to be on u/v points.

`do j=js,je ; do I=Isq,Ieq`
Isn't this loop at u-points?

`do J=Jsq,Jeq ; do i=is,ie`
and this at v-points?

Reply:
I went and looked at the definitions of each of the indices (instead of just relying on faulty memory). Long story short, those loop indices are correct.

The `Isq` and `Jsq` variables are assigned based off the `IscB` and `JscB` indices which if you were looping over both would mean that you're at the top-right vertex of the cell, using only one of them as a loop index means that you're at the velocity point.